packages/grafana-data/src/datetime/rangeutil.ts

```diff
@@ -58,6 +60,10 @@ const rangeOptions: TimeOption[] = [
   { from: 'now-1y', to: 'now', display: 'Last 1 year' },
   { from: 'now-2y', to: 'now', display: 'Last 2 years' },
   { from: 'now-5y', to: 'now', display: 'Last 5 years' },
+  { from: 'now/fQ', to: 'now', display: 'This fiscal quarter so far' },
```

Comment:
@marefr We want to add `f` as a way to round to fiscal quarters/years. However I've noticed that these strings gets sent to the backend. My first guess would be for alerting? I guess we need to implement the same parsing on the back end?